steward/src/cellars/adaptors/aave_v2.rs

```diff
@@ -0,0 +1,293 @@
+use abscissa_core::tracing::log::debug;
+use ethers::{abi::AbiEncode, types::Bytes};
+use steward_abi::{
+    aave_a_token_adaptor::AaveATokenAdaptorCalls,
+    aave_a_token_adaptor_v2::AaveATokenAdaptorV2Calls,
+    aave_debt_token_adaptor::AaveDebtTokenAdaptorCalls,
+    aave_debt_token_adaptor_v2::AaveDebtTokenAdaptorV2Calls,
+};
+use steward_proto::steward::{
+    aave_a_token_adaptor_v1, aave_a_token_adaptor_v2, aave_debt_token_adaptor_v1,
+    AaveATokenAdaptorV1Calls, AaveDebtTokenAdaptorV1Calls,
```

Comment:
I had decided that for adaptors whose V1 was now blocked, I'd just leave it unversioned, but you're right it's inconsistent. Fixed that.

As far as the name overlapping, I think it would be a good tech debt issue. There are a lot of places where I didn't use an in-line import path and you can't immediately tell which type it is.

Reply:
Update: I made all of the ABI file and type names consistent with the versioning, and then aliased the imports with the `Abi*` prefix.